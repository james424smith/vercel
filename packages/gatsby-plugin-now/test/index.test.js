test('test generated now routes', async () => {
  const nowRoutes = require('./fixtures/public/__now_routes_g4t5bY.json');

  expect(nowRoutes).toMatchSnapshot();
});
