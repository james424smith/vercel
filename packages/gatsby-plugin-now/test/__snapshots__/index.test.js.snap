// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test generated now routes 1`] = `
Array [
  Object {
    "headers": Object {
      "Location": "/",
    },
    "src": "/my-special-redirect",
    "status": 302,
  },
  Object {
    "handle": "filesystem",
  },
  Object {
    "headers": Object {
      "Location": "/page-2",
    },
    "src": "/page2",
    "status": 301,
  },
  Object {
    "headers": Object {
      "Location": "/page-2/",
    },
    "src": "/page2/",
    "status": 301,
  },
  Object {
    "headers": Object {
      "Location": "/",
    },
    "src": "/orange",
    "status": 302,
  },
  Object {
    "headers": Object {
      "Location": "/",
    },
    "src": "/grape",
    "status": 302,
  },
  Object {
    "headers": Object {
      "Location": "/page-2/",
    },
    "src": "/blue",
    "status": 302,
  },
  Object {
    "headers": Object {
      "Location": "/page-2/",
    },
    "src": "/randirect",
    "status": 302,
  },
  Object {
    "headers": Object {
      "Location": "/",
    },
    "src": "/juice",
    "status": 302,
  },
  Object {
    "headers": Object {
      "Location": "/",
    },
    "src": "/soda",
    "status": 302,
  },
  Object {
    "headers": Object {
      "Location": "/page-2/",
    },
    "src": "/donut",
    "status": 302,
  },
  Object {
    "headers": Object {
      "Location": "/page-2/",
    },
    "src": "/randorect",
    "status": 302,
  },
]
`;
